python/setup.py

```diff
@@ -194,13 +210,16 @@ class bdist(custom_opts, _bdist.bdist):
     have_wheel = False
     try:
         import wheel.bdist_wheel as _bdist_wheel
+
         class bdist_wheel(custom_opts, _bdist_wheel.bdist_wheel):
             parent = _bdist_wheel.bdist_wheel
             user_options = _bdist_wheel.bdist_wheel.user_options + custom_user_options
+
         have_wheel = True
     except: None
 
-    class build_ext(_build_ext.build_ext):
+    class build_py(_build_py.build_py):
```

Comment:
Maybe I'm mistaken, but looks like we don't need to derive from`build_py` anymore, and `build_ext` only is sufficient.